File: archive/challenge-0001/easy.rb
#!/usr/bin/env ruby
# http://www.reddit.com/r/dailyprogrammer/comments/pih8x/
arr = ["name : ","age : ","redditname : "].collect{|e| print e; $stdin.gets.chomp!}
puts "your name is %s, you are %s years old, and your username is %s" % arr

File: archive/challenge-0001/easy.cxx
#include <iostream>
#include <string>
using std::cout;
using std::cin;
using std::endl;
using std::string;

int main() {
  string name;
  string age;
  string redditname;

  cout << "enter name: ";
  std::getline (cin, name);

  cout << "enter age: ";
  std::getline (cin, age);

  cout << "enter reddit name: ";
  std::getline (cin, redditname);

  cout << "your name is " << name
       << " you are " << age
       << " old, and your username is "
       << redditname << endl;

  return 0;
}
